enterprise/server/testutil/enterprise_testenv/enterprise_testenv.go

```diff
@@ -37,6 +38,10 @@ func GetCustomTestEnv(t *testing.T, opts *Options) *testenv.TestEnv {
 		cache := redis_cache.NewCache(redisClient)
 		env.SetCache(cache)
 	}
+
+	if flag.Lookup("app.enable_write_to_olap_db") != nil {
+		flags.Set(t, "app.enable_write_to_olap_db", opts.EnableWriteToOLAPDB)
```

Comment:
nit: you can also write `flags.Set(t, "app.enable_write_to_olap_db", true)` in `NewRBETestEnv(t)` to avoid introducing another field in `Options`